### DISCUSSION TITLE
upgrade gitx

### DISCUSSION THREAD

**Discussion by LiJohnson** [Homebrew/discussions#4250] [Casks]
### Output of `brew config`

```shell
none
```

### Output of `brew doctor`

```shell
none
```

### Description of issue

homebrew casks [gitx](https://github.com/Homebrew/homebrew-cask/blob/master/Casks/gitx.rb) is an old version.
I found out that,gitx now release here https://github.com/gitx/gitx

### Relevant casks

gitx

**Comment by SMillerDev** [comment]
https://docs.brew.sh/Updating-Software-in-Homebrew
